backend/pps/sch.py

```diff
@@ -1,27 +1,28 @@
 import math
 
-from ffxivdpscalc.calc import CharacterStats
+from backend.calc import CharacterStats
+from backend.pps.pps import HealerPps
 
 
-class SchCharacterStats(CharacterStats):
+class SchPps(HealerPps):
     r2_potency = 200
     b3_potency = 290
     bio_potency = 70
     ed_potency = 100
-
-    def __init__(self, job, wd, mainstat, det, crit, dh, speed, ten, pie):
-        super().__init__(job, wd, mainstat, det, crit, dh, speed, ten, pie)
+        
+    def get_pps(self, character_stats, caster_tax):
+        return self.total_potency_spreadsheet_port(character_stats, caster_tax) / self.get_cycle(caster_tax)
         
     # hard coded for a ~180 second cycle, actual length calculated by get_cycle
     # todo: extend this for variable length
-    def pps_spreadsheet_port(self, caster_tax, num_ed_per_min):
+    def total_potency_spreadsheet_port(self, character_stats, caster_tax):
         # do as the spreadsheet do        
-        short_gcd = self.get_gcd()
+        short_gcd = character_stats.get_gcd()
         
         result = 0
-        result += 3*self.ed_potency*num_ed_per_min
+        result += 12*self.ed_potency
```

Comment:
reee why do you hate spaces? 

allowing it for now but i'm getting the linter tomorrow. kek.

Reply:
actually screw it i found another change so i'm gonna nitpick you on style too

Reply:
put the ed per min arg back here